<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Keyboard Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QKeySequenceEdit" name="ShortCut1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="keySequence">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ServerStartButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopServerButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ShortCut2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="keySequence">
     <string/>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ShortCut3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ShortCut4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ShortCut5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="ShortCut6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW1">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>5</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW2">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>55</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 2 </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW3">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>105</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW4">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>155</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW5">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>205</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSW6">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>255</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>switch 6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>323</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PortInputLbl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>304</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This server allow you to simulate key presses via a serial input. Currently only the letters a to z keys can be simulated (from a azerty keyboard). Your device's baud rate must be 9600, and this server will react to int between 1 and 6 sent on the serial port of your choice.&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Restart the server after changing any shortcuts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutTerm"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed1"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed3"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed4"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed5"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutLed6"/>
   </widget>
   <widget class="QLabel" name="labelServerStatus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelHold">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>2</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>22</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>72</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>122</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>172</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>222</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSw6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>272</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Filz">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
   </widget>
   <addaction name="menu_Filz"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
